hedera-node/src/main/java/com/hedera/services/txns/validation/OptionValidator.java

```diff
@@ -60,6 +60,12 @@
 	ResponseCodeEnum tokenSymbolCheck(String symbol);
 	ResponseCodeEnum tokenTransfersLengthCheck(List<TokenTransferList> tokenTransferLists);
 
+	ResponseCodeEnum nftMetadataCheck(byte[] metadata);
+	ResponseCodeEnum maxBatchSizeMintCheck(int length);
+	ResponseCodeEnum maxBatchSizeWipeCheck(int length);
+	ResponseCodeEnum maxBatchSizeBurnCheck(int length);
+	ResponseCodeEnum maxQueryRangeCheck(int start, int end);
```

Comment:
I think that a better name would be `nftMaxQueryRangeCheck`. If we do not include NFT, it might be confusing